evansmusomi/react-native-102
# Adds tabbed navigation, find and share place screens

*What this does*
- Adds tabbed navigation with icons
- Adds FindPlace and SharePlace screens

*How to verify*
- Click the login button on the home screen
- This should navigate to a screen with two tabs below labelled "Find Place" and "Share Place"
- Clicking on either tab should change the text on the screen and the title in the navbar
